test/e2e/jest-setup/docker.ts

```diff
@@ -46,18 +44,10 @@ export const setupPostgresContainer = async (
 
   // Uncomment the following lines if you are going to populate the db with a snapshot
 
-  // await container.putArchive(path.join(__dirname, 'db-snapshot.tar'), {
```

Comment:
Maybe this can be retained using and ENV switching between the two modes? I'm just concerned we'll lose the update logic

Reply:
this is a feature to easily pre-load testing db with some sort of fixture or snapshot data. I'm not sure it might be super useful here.

Reply:
If we're expecting this to be an ongoing action when the data or schema changes, would documentation in the Wiki suffice?

Reply:
What I mean, this is not a feature we might need for this project. It was taken from Rosetta (we used the same project as it would be familiar for several adrestia products) because we wanted to preload a `mainnet` or `testnet` `db-sync` dbs but I'm not sure if snapshot testing is needed here as the data can be easily populated.